src/main/java/database/exception/handler/FileDatabaseExceptionHandler.java

```diff
@@ -0,0 +1,44 @@
+package database.exception.handler;
+
+import database.exception.exception.BadRequestException;
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+import io.javalin.http.NotFoundResponse;
+
+import java.util.Map;
+
+public class FileDatabaseExceptionHandler {
+
+    public static void register(Javalin app) {
+        app.exception(BadRequestException.class, FileDatabaseExceptionHandler::handleBadRequest);
+        app.exception(NotFoundResponse.class, FileDatabaseExceptionHandler::handleNotFound);
+        app.exception(Exception.class, FileDatabaseExceptionHandler::handleInternalServerError);
+    }
+
+    private static void handleBadRequest(BadRequestException e, Context ctx) {
+        ctx.status(400); // Set status code
+        ctx.json(Map.of(
+                "status", 400,
+                "error", "Bad request",
+                "message", e.getMessage()
+        ));
```

Comment:
Since we have the same structure for all error responces, consider creating class `ErrorResponse` with all these fields.